include/yaramod/utils/modifying_visitor.h

```diff
@@ -10,6 +10,34 @@
 #include "yaramod/utils/visitor.h"
 
 namespace yaramod {
+/**
+ * This class captures current TokenStream and first and last TokenIi of the expression which it is given
+ * in it's constructor. This data can be very useful at the end of the visit method of ModifyingVisitors
+ * by the cleanUpTokenStreams function.
+ */
+class TokenStreamContext {
```

Comment:
Please follow code style used in other files

Reply:
I added a new line in between class name and the curly bracket and also one more new line before `private` keyword. I hope it is alright now.